src/test/java/com/google/firebase/auth/multitenancy/FirebaseTenantClientTest.java

```diff
@@ -68,6 +69,7 @@
 
   private static final String TENANTS_BASE_URL = PROJECT_BASE_URL + "/tenants";
 
+
```

Comment:
Please add more tests for `FirebaseUserManager.java` and `FirebaseTenantClientTest.java` For eg: Check that the right endpoint is being called, when the environment variable is set (or unset).

Here is a [link](https://github.com/firebase/firebase-admin-node/blob/f624b2963900deebed0fdb5cbe66ce7eb0d244b8/test/unit/auth/auth-api-request.spec.ts#L910) to the tests in the Node.js SDK, for reference :)